src/components/UI/Slider/SliderWithMarkers.vue

```diff
@@ -83,6 +94,37 @@ export default {
     },
   },
   methods: {
+    /**
+     * Update the content and positioning of a marker tooltip
+     * @param {Number} markerIndex - The index of the marker
+     */
+    updateMarkerTooltip(markerIndex) {
+      if (markerIndex == null) {
+        // don't show anything
+        this.markerTooltipContent = null;
+      } else {
+        // convert the time in seconds to ISO time
+        let timeInSeconds = this.localMarkerPositions[markerIndex];
+        let ISOTimeObject = convertSecondsToISOTime(timeInSeconds);
+        this.markerTooltipContent =
+          ISOTimeObject.minute +
+          ":" +
+          ISOTimeObject.second +
+          ":" +
+          ISOTimeObject.millisecond;
```

Comment:
a cleaner one might this:

```suggestion
        this.markerTooltipContent = `${ISOTimeObject.minute}:${ISOTimeObject.second}:${ISOTimeObject.millisecond}`
```

and then, can apply this for the string concatenations below